datajoint/connection.py

```diff
@@ -59,7 +59,7 @@ def __init__(self, host, user, passwd, init_fun=None):
             port = int(port)
         else:
             port = config['database.port']
-        self.conn_info = dict(host=host, port=port, user=user, passwd=passwd)
+        self.conn_info = dict(host=host, port=port, user=user, passwd=passwd, max_allowed_packet=1024**3) # 1073741824
```

Comment:
Yeah, I'm also curious what this does? I'm guessing that you have encountered a problem with a large query hitting the packet size limit?